packages/pluggableWidgets/slider-web/src/utils/getMaxValue.ts

```diff
@@ -0,0 +1,15 @@
+import { SliderContainerProps } from "../../typings/SliderProps";
+
+export function getMaxValue(props: SliderContainerProps): number {
+    switch (props.maxValueType) {
+        case "static": {
+            return Number(props.staticMaximumValue);
+        }
+        case "dynamic": {
+            return Number(props.maxAttribute?.value ?? 100);
+        }
+        case "expression": {
+            return Number(props.expressionMaximumValue?.value ?? 100);
```

Comment:
- Do these two 100 values have the same meaning? If so, extract it into a const.
- For `getMinValue`, `getMaxValue` and `getStepValue`, this means that while the values are still loading, it'll show 100. Is that per requirements cuz it sounds a bit weird?